Reset_Bjoerns_Password/README.md

```diff
@@ -0,0 +1,47 @@
+# Reset Bjoern's Password
+
+## Type
+
+Broken Authentication
+
+## Description
+
+Reset the password of **Bjoern's internal account** via the **Forgot Password** mechanism with the original answer to his security question.

```

Comment:
nice one. very concise but yet contains all relevant information. 
Here, you should also add something for the 'how are you gonna do that?' part, like using OSINT strategies, like search engines to retrieve a bit of interesting information about bjoern which might help answering the security question.

also, you could mention the original security question in your documentation, otherwise readers might be confused about some steps later where you non-obviously try the zip code as answer

Reply:
I have the images in the README :D